src/v2/Components/__tests__/AuctionBuyersPremiumDialog.jest.tsx

```diff
@@ -76,5 +76,19 @@ describe("AuctionBuyersPremiumDialog", () => {
         "On the portion of the hammer price in excess of $2,500,000: 12%"
       )
     })
+
+    describe("with a percentage that isn't a whole number", () => {
```

Comment:
I would say to avoid adding more enzyme tests ([more info](https://github.com/artsy/force/blob/bb6cc363dfd11c71967e7b8767fe39d168025ffb/docs/best_practices.md#write-unit-tests-for-new-components)), we recommend using react testing library instead of adding more tests with the old approach.
[example](https://github.com/artsy/force/blob/bb6cc363dfd11c71967e7b8767fe39d168025ffb/src/v2/Apps/Settings/Routes/Saves/Components/__tests__/SettingsSavesArtists.jest.tsx)

Reply:
Here's the latest best practices docs that cover the change: https://github.com/artsy/force/blob/master/docs/best_practices.md#write-unit-tests-for-new-components

Reply:
Going forward, should I be converting any test files I come across?

Reply:
if it's not too complicated and the time is ok I would say yes, otherwise just to not increase the number of tests with enzyme use RTL